### PR TITLE
Use licenseRefLookup to normalize license expression

### PR DESCRIPTION
**Pull request by qtomlinson**

### CODE REVIEW

**Review by ljones140 — COMMENTED** [review]
Good to see the recursion removed in favour of the SPDX package.

Happy to approve.

Do we need to issue a release for the package and update it in the service first?

As releases are tied to the GH package

**Comment by qtomlinson** [comment]
@ljones140 Thanks for the feedback!  
https://github.com/clearlydefined/spdx-expression-parse.js/pull/9 has to be merged first, then https://github.com/clearlydefined/spdx/pull/34 can be merged and released.

I will update this PR to point to the updated SPDX.

File: lib/utils.js [hunk 1]
```diff
@@ -416,49 +416,22 @@ function joinExpressions(expressions) {
...
+  const licenseVisitor = licenseExpression =>
+    scancodeMap.get(licenseExpression) || SPDX.normalizeSingle(licenseExpression)
+  const parsed = SPDX.parse(rawLicenseExpression, licenseVisitor, licenseRefLookup)
+  const result = SPDX.stringify(parsed)
   if (result === 'NOASSERTION') logger.info(`ScanCode NOASSERTION from ${rawLicenseExpression}`)
 
   return result
 }
 
-function _normalizeParsedLicenseExpression(parsedLicenseExpression, logger) {
```
Comment (elrayle) [inline lib/utils.js:440]: Really good to see this move into the parser.  Thanks for making this happen!

**Review by elrayle — APPROVED** [review]
Thanks to @qtomlinson and @yashkohli88 for all their work to push the scancode LicenseRef code through to completion!!!

**Pull request merged**